ksqldb-functional-tests/src/test/java/io/confluent/ksql/test/model/SchemaNodeTest.java

```diff
@@ -49,12 +49,12 @@ public void shouldRoundTrip() {
   }
 
   @Test
-  public void shouldRoundNoKeyFormat() {
+  public void shouldRoundTripNoKeyFormat() {
```

Comment:
Changes in this file are related to review comment in https://github.com/confluentinc/ksql/pull/6365